core/evm.go

```diff
@@ -31,7 +31,7 @@ type ChainContext interface {
 	// Engine retrieves the chain's consensus engine.
 	Engine() consensus.Engine
 
-	// GetHeader returns the hash corresponding to their hash.
+	// GetHeader returns the header corresponding to their hash.
```

Comment:
I think "their hash" is a bit odd too, and also there are two args, not only one
```suggestion
	// GetHeader returns the header corresponding to the hash/number argument pair.
```